core/unit_test/TestSharedSpace.cpp

```diff
@@ -1,7 +1,14 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 // SPDX-FileCopyrightText: Copyright Contributors to the Kokkos project
 #include <gtest/gtest.h>
+#include <Kokkos_Macros.hpp>
+#ifdef KOKKOS_ENABLE_EXPERIMENTAL_CXX20_MODULES
+import kokkos.core;
+import kokkos.core_impl;
+#else
 #include <Kokkos_Core.hpp>
+#endif
+#include <impl/Kokkos_ClockTic.hpp>
```

Comment:
should these get a FIXME comment or is it apparent enough from the context?

Reply:
I'm not sure if there really is something to fix (if we have a clear preference for including `Impl` headers over importing `kokkos.core_impl`). In a discussion with @dalg24, he preferred including impl headers (that are pretty much self-contained).